Kickstarter-iOS/DataSources/ProjectPageViewControllerDataSource.swift

```diff
@@ -20,21 +22,48 @@ internal final class ProjectPageViewControllerDataSource: ValueCellDataSource {
 
   // TODO: Internationalize strings
   private enum HeaderValue: String {
+    case overview = "Overview"
     case environmentalCommitments = "Environmental commitments"
     case faqs = "Frequently asked questions"
     case risks = "Risks and challenges"
   }
 
   func load(
     navigationSection: NavigationSection,
-    projectProperties: ExtendedProjectProperties,
+    project: Project,
```

Comment:
Unfortunately had to revert this back to the `Project` parameter because the overview needs it.